test/e2e/specs/editor/blocks/navigation.spec.js

```diff
@@ -0,0 +1,59 @@
+/**
+ * WordPress dependencies
+ */
+const {
+	test,
+	expect,
+	Editor,
+} = require( '@wordpress/e2e-test-utils-playwright' );
+
+test.use( {
+	editor: async ( { page }, use ) => {
+		await use( new Editor( { page, hasIframe: true } ) );
+	},
+} );
+
+test.describe(
+	'As a user I want the navigation block to fallback to the best possible default',
+	() => {
+		test.beforeAll( async ( { requestUtils } ) => {
+			await requestUtils.activateTheme( 'emptytheme' );
+		} );
+
+		test.beforeEach( async ( { admin } ) => {
+			await admin.createNewPost();
+		} );
+
+		test.afterAll( async ( { requestUtils } ) => {
+			await requestUtils.activateTheme( 'twentytwentyone' );
+		} );
+
+		test.afterEach( async ( { requestUtils } ) => {
+			await requestUtils.deleteAllPosts();
+		} );
+
+		test( 'default to a list of pages if there are no menus', async ( {
+			editor,
+		} ) => {
+			await editor.insertBlock( { name: 'core/navigation' } );
+
+			const pageListBlock = editor.canvas.getByRole( 'document', {
+				name: 'Block: Page List',
+			} );
+
+			await expect( pageListBlock ).toBeVisible( {
+				// Wait for the API to be loaded.
+				timeout: 10000,
```

Comment:
Ah right so you're happy with this approach of bumping up the timeout.

I much prefer the approach of not waiting on the implementation details (i.e. network request). Testing user perceivable things is much much better.

Reply:
oh, interesting, won't this delay the github actions when they run for each commit? I guess it's not too long, but I didn't know this was accepted practice

Reply:
I guess it will only wait for the block to appear. Which is the same as waiting on the network request.

Reply:
that's smart then, good to know!

Reply:
Yeah, the default timeout is 5 seconds, we're increasing the timeout to give the network requests more time to resolve. It will pass as soon as the page list block is visible.